adoc/admin-security-certificates.adoc

```diff
@@ -2,31 +2,49 @@
 
 During the installation of {productname}, a CA (Certificate Authority) certificate is generated,
 which is then used to authenticate and verify all communication. This process also creates
-and distributes client certificates for the components.
+and distributes client and server certificates for the components.
 
 == Communication Security
 Communication is secured with TLS v1.2 using the AES 128 CBC cipher.
-All client certificates are 4096 Bit RSA encrypted.
+All certificates are 2048 bit RSA encrypted.
 
 == Certificate Validity
 The CA certificate is valid for 3650 days (10 years) by default.
-Client certificates are valid for 365 days (1 year) by default.
+Client and server certificates are valid for 365 days (1 year) by default.
 
 == Certificate Location
-Certificates are located under `/etc/kubernetes/pki` on all master nodes.
-Three root CA required for {productname} are:
+Required CAs for {productname} are stored on all master nodes:
 
-* {kube} apiserver: `/etc/kubernetes/pki/ca.{crt.key}`
-* Front-end proxy: `/etc/kubernetes/pki/front-proxy-ca.{crt.key}`
-* Etcd: `/etc/kubernetes/pki/etcd/ca.{crt.key}`
+[%header,cols=3*]
+|===
+|Common Name
+|Path
+|Description
+
+|kubernetes
+|/etc/kubernetes/pki/ca.crt,key
+|kubernetes general CA
+
+|etcd-ca
+|/etc/kubernetes/pki/etcd/ca.crt,key
+|Etcd cluster
+
+|kubelet-ca
+|/var/lib/kubelet/pki/ca.crt,key
+|Kubelet components
```

Comment:
For me it's not ca.crt,key but kubelet-ca.crt,key.

Am I wrong ?

Reply:
Thanks, my fault, typo.

Reply:
#620